scripts/action-entrypoint-generator/action-entrypoint-generator

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+"""
+Generate the entrypoint.sh files for our GitHub actions.
+"""
+import argparse
+import sys
+from pathlib import Path
+
+
+TEMPLATE = """#!/usr/bin/env bash
+set -euo pipefail
+# AUTOGENERATED FILE, DO NOT EDIT!
```

Comment:
This line is not in-sync with the entrypoints committed ?

Reply:
Indeed, I updated the comments here but did not regenerate the scripts. Fixed now.